frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -57,6 +57,24 @@ interface NewRunParametersProps {
   handleParameterChange?: (parameters: RuntimeParameters) => void;
 }
 
+const inputConverter = (paramStr: string, paramTypeIdx: number) => {
+  if (ParameterType_ParameterTypeEnum[paramTypeIdx] === 'BOOLEAN' && paramStr === 'True') {
```

Comment:
I tend to think using switch statement here can help improve the readability: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch.

For example:
```
switch (paramType) {
  case ParameterType_ParameterTypeEnum.BOOLEAN:
       // do something
       break;
  case ParameterType_ParameterTypeEnum.NUMBER_INTEGER:
       // do something
       break;
  ...
  default:
       console.warn('Unknown parameter type: ' + paramType);
       return undefined;
}
```